mmdet3d/models/backbones/multi_backbone.py

```diff
@@ -0,0 +1,122 @@
+import copy
+import torch
+from mmcv.cnn import ConvModule
+from mmcv.runner import load_checkpoint
+from torch import nn as nn
+
+from mmdet.models import BACKBONES, build_backbone
+
+
+@BACKBONES.register_module()
+class MultiBackbone(nn.Module):
+    """MultiBackbone with different config.
+
+    Args:
+        num_streams (int): The number of backbones.
+        backbones (list or dict): A list of backbone configs.
+        aggregation_mlp_channels (list[int]): Specify the mlp layers
+            for feature aggregation.
+        conv_cfg (dict): Config dict of convolutional layers
+        norm_cfg (dict): Config dict of normalization layers
+        act_cfg (dict): Config dict of activation layers
+        suffixes (list): A list of suffixes to rename the return dict
+            for each backbone.
+    """
+
+    def __init__(self,
+                 num_streams,
+                 backbones,
+                 aggregation_mlp_channels=None,
+                 conv_cfg=dict(type='Conv1d'),
+                 norm_cfg=dict(type='BN1d', eps=1e-5, momentum=0.01),
+                 act_cfg=dict(type='ReLU'),
+                 suffixes=('net0', 'net1'),
+                 **kwargs):
+        super().__init__()
+        assert isinstance(backbones, dict) or isinstance(backbones, list)
```

Comment:
Test these assertions in line 36, 43, and 44 by passing wrong arguments with `with pytest.raises(AssertionError)`.